### ISSUE TITLE
chroma-diagnostics is truncating log files

### ISSUE THREAD

**Issue by brianjmurrell** [intel-hpdd/intel-manager-for-lustre#259]
As an example, looking at this _chroma-diagnostics_ [collection](http://jenkins.lotus.hpdd.lab.intel.com/job/integration-tests-shared-storage-configuration/1139/arch=x86_64,distro=el7/artifact/test_logs/diagnostics_lotus-49vm17_20170831T072556/):

```
￼chroma-agent.log		0 B
￼chroma-agent-console.log	0 B
```

There is no way these two files above should be 0 bytes.

```
￼messages	256.00 KB	￼ view
```
￼
And there is no way that the above should only be 256KB.

**Comment by brianjmurrell** [comment]
[Another](http://jenkins.lotus.hpdd.lab.intel.com/job/integration-tests-shared-storage-configuration/1152/arch=x86_64,distro=el7/artifact/test_logs/diagnostics_lotus-52vm17_20170901T005635/:

```
messages	256.00 KB
```

**Comment by brianjmurrell** [comment]
[Another](http://jenkins.lotus.hpdd.lab.intel.com/job/integration-tests-shared-storage-configuration/arch=x86_64,distro=el7/1135/artifact/test_logs/diagnostics_lotus-57vm17_20170831T082222/):

```
messages	256.00 KB
```

**Comment by brianjmurrell** [comment]
[Another](http://jenkins.lotus.hpdd.lab.intel.com/job/integration-tests-shared-storage-configuration/arch=x86_64,distro=el7/1143/artifact/test_logs/diagnostics_lotus-44vm7_20170831T161543/)

This issue is making it difficult to determine which issue a failed run is without a full _messages_ file.

**Comment by jgrund** [comment]
Hrrm, something looks similar between all of those.

**Comment by brianjmurrell** [comment]
[Another](http://jenkins.lotus.hpdd.lab.intel.com/job/integration-tests-shared-storage-configuration/arch=x86_64,distro=el7/1164/artifact/test_logs/diagnostics_lotus-49vm17_20170901T193509/):
```
chroma-agent.log		0 B
chroma-agent-console.log	0 B
```

**Comment by brianjmurrell** [comment]
[Another](http://jenkins.lotus.hpdd.lab.intel.com/job/integration-tests-shared-storage-configuration/arch=x86_64,distro=el7/1165/artifact/test_logs/diagnostics_lotus-48vm17_20170901T200423/)

**Comment by brianjmurrell** [comment]
[Another](http://jenkins.lotus.hpdd.lab.intel.com/job/integration-tests-shared-storage-configuration/arch=x86_64,distro=el7/1210/artifact/test_logs/diagnostics_lotus-9vm17_20170905T193301/)

**Comment by brianjmurrell** [comment]
[Another](http://jenkins.lotus.hpdd.lab.intel.com/job/integration-tests-shared-storage-configuration/arch=x86_64,distro=el7/1216/artifact/test_logs/diagnostics_lotus-57vm17_20170905T172554/)

**Comment by brianjmurrell** [comment]
[Another](http://jenkins.lotus.hpdd.lab.intel.com/job/integration-tests-shared-storage-configuration/arch=x86_64,distro=el7/1230/artifact/test_logs/diagnostics_lotus-44vm17_20170906T192715/)

**Comment by brianjmurrell** [comment]
[Another](http://jenkins.lotus.hpdd.lab.intel.com/job/integration-tests-shared-storage-configuration/arch=x86_64,distro=el7/1243/artifact/test_logs/diagnostics_lotus-48vm18_20170907T195720/)

**Comment by brianjmurrell** [comment]
[Another](http://jenkins.lotus.hpdd.lab.intel.com/job/integration-tests-shared-storage-configuration/arch=x86_64,distro=el7/1246/artifact/test_logs/diagnostics_lotus-44vm7_20170907T205813/)

**Comment by jgrund** [comment]
Fixed by #267
